README.md

```diff
@@ -239,10 +243,55 @@ qiime evident alpha-effect-size-by-category \
     --o-effect-size-results alpha_effect_sizes.qza
 ```
 
-## Help with evident
+## Repeated Measures
 
-If you encounter a bug in evident, please post a GitHub issue and we will get to it as soon as we can.
-We welcome any ideas or documentation updates/fixes so please submit an issue and/or a pull request if you have thoughts on making evident better.
+Evident supports limited analysis of repeated measures.
+When your dataset has repeated measures, you can calculate `eta_squared` for alpha diversity differences.
+Note that only alpha diversity is supported with repeated measures.
```

Comment:
What about log ratios?

Reply:
Log-ratios can be used my only issue is that the wording would have to change a good amount both here and in the manuscript. I'm not opposed in principle but IMO it can wait until later. Let me know if you think it's important enough to warrant a more immediate update.

Reply:
hmm... would it suffice to here and in the manuscript put a parenthetical `alpha diversity (or other univariate measures such as log ratios)`? @ElDeveloper, I think you brought the comment up about log ratio, what do you think?

Reply:
@gibsramen, If I understand correctly log ratios are not supported yet i.e. the semantic type in the plugin still indicates only a variant for alpha diversity. So the path forward is to add support for that in another PR *AND* then update the documentation? I think that makes sense.

Reply:
Yeah that should work. I think noting the possibility in the manuscript and updating the documentation should be fine. We'll take care of that in another PR.